.github/workflows/ci-weekly.yml

```diff
@@ -1,74 +1,164 @@
-name: CI - FAST Validation
+name: CI - FAST
 
 on:
-  schedule:
-  - cron: 0 7 * * 3
-
+  push:
+    branches:
+    - releases/*
+  pull_request:
+    branches:
+    - master
+    - releases/*
+    - features/*
 
 jobs:
-  build_windows:
-    name: Validate on Windows
-    runs-on: windows-latest
-
+  build_linux:
+    runs-on: [ubuntu-latest]
     steps:
     - uses: actions/checkout@v2
+    - name: Get yarn cache directory path
+      id: yarn-cache-dir-path
+      run: echo '::set-output name=dir::$(yarn cache dir)'
 
-    - name: Setup Node.js environment
-      uses: actions/setup-node@v2.1.0
+    - name: Set up node_modules cache
```

Comment:
add a line break between `uses:..` and `- name:...`

Reply:
done